include/aspect/material_model/grain_size.h

```diff
@@ -400,6 +400,12 @@ namespace aspect
         bool use_enthalpy;
         bool use_bilinear_interpolation;
 
+        /**
```

Comment:
I would move all of this to a different place in the code, i.r. around line 186 which says "Parameters controlling the grain size evolution." Here, it's placed inside a section that is about reading in material files (the enum below belongs with the section above, since both of these are about "properties of the material files we read in".